cmd/simulacra/simulacra.go

```diff
@@ -204,9 +217,61 @@ func getConfigFromYaml(configPath string) (*Config, error) {
 	return &config, nil
 }
 
+// Parse the metadata image name with format 'projects/debian-cloud/global/images/debian-11-bullseye-v20230711'
```

Comment:
How certain are we that the image will always be in this format? Have we looked at any custom images to make sure they follow this format too?

Reply:
Discussed offline. What we ended up doing was if the image name in the metadata file does not have the prefix "/projects", then we ask the user (using command line input) for the image family.